lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -42,6 +44,8 @@ sub init {
     $self->{ClonePartitionsDialog}         = Installation::Partitioner::LibstorageNG::v4_3::ClonePartitionsDialog->new({app => YuiRestClient::get_app()});
     $self->{CreatePartitionTablePage}      = Installation::Partitioner::LibstorageNG::v4_3::CreatePartitionTablePage->new({app => YuiRestClient::get_app()});
     $self->{DeletingCurrentDevicesWarning} = Installation::Partitioner::LibstorageNG::v4_3::DeletingCurrentDevicesWarning->new({app => YuiRestClient::get_app()});
+    $self->{ErrorDialog}                   = Installation::Partitioner::ErrorDialog->new({app => YuiRestClient::get_app()});
```

Comment:
I'm not sure if we should put it in the root directory, as REST API is not available in SP0, or SLE 12.

Reply:
Changed.